File: src/main/java/thedarkcolour/futuremc/enchantment/EnchantmentMultishot.java
package thedarkcolour.futuremc.enchantment;

import net.minecraft.enchantment.Enchantment;

public class EnchantmentMultishot extends EnchantmentCrossbowBase {
    public EnchantmentMultishot() {
        super(Rarity.RARE, "multishot");
    }

    @Override
    public int getMinEnchantability(int enchantmentLevel) {
        return 20;
    }

    @Override
    public int getMaxLevel() {
        return 1;
    }

    @Override
    protected boolean canApplyTogether(Enchantment ench) {
        return super.canApplyTogether(ench) && ench != Enchantments.PIERCING;
    }
}

File: src/main/java/thedarkcolour/futuremc/enchantment/EnchantmentMultishot.kt
package thedarkcolour.futuremc.enchantment

import net.minecraft.enchantment.Enchantment
import thedarkcolour.futuremc.registry.FEnchantments

class EnchantmentMultishot : EnchantmentCrossbowBase(Rarity.RARE, "multishot") {
    override fun getMinEnchantability(enchantmentLevel: Int): Int {
        return 20
    }

    override fun getMaxLevel(): Int {
        return 1
    }

    override fun canApplyTogether(ench: Enchantment): Boolean {
        return super.canApplyTogether(ench) && ench !== FEnchantments.PIERCING
    }
}